src/building_blocks/ectf.md

```diff
@@ -0,0 +1,4 @@
+# ECtF
+
+This protocol allow the User and the Notary to convert their shares of an ECDH secret into shares of the pre-master secret (PMS).
```

Comment:
allows

Reply:
actually, should be "enables". old habits :)